client/blocks/nps-survey/docs/example.jsx

```diff
@@ -0,0 +1,71 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import NpsSurvey from '../';
+import { isEnabled } from 'config';
+
+class NpsSurveyExample extends PureComponent {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			wasDismissed: false,
+			wasSubmitted: false,
+			surveyName: null,
+			recommendationValue: null,
+		};
+		this.handleDismissed = this.handleDismissed.bind( this );
+	}
+
+	handleDismissed( event ) {
+		this.setState( {
+			wasDismissed: true,
+			wasSubmitted: event.wasSubmitted,
+			surveyName: event.surveyName,
+			recommendationValue: event.recommendationValue
```

Comment:
We are in control of the sending of the event, so I don't think that is necessary. We should guarantee when we send the event that it is an int.